docs/DeveloperGuide.adoc

```diff
@@ -809,6 +811,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`***` |user | add appointments to a calendar | schedule different appointments without time clash
+
+|`**` |user | change the size of different windows of the address book application UI | it looks pleasing to the eyes
+
+|`**` |user who contacts different parties involved in an appointment | draft the message about appointment details automatically | don’t have to do it
+
+|`**` |user who frequently contacts certain people | sort the contacts by contacting frequency |  find those people I frequently contact easily
+
+|`***` |user | see the location of a given appointment |  plan for travel
+
+|`*` |user who is time conscious |  see the time and date |  plan schedules faster
+
+|`*` |user | output the contacts to a separate list |  have a backup copy of the contacts
+
+|`***` |user | remove appointments to a calendar |  keep my scheduling neat and tidy
+
+|`***` |user | edit appointment timings |  change the time of the appointment when I need to re-schedule
+
+|`***` |user | change appointment location |  change the venue if the original one is not available
+
+|`***` |user | change people involved in appointment |  add new people involved or remove people no longer involved
+
+|`***` |user | have notifications or reminder for urgent upcoming events that I need to take note of |  set my priorities straight
+
+|`**` |user | change the colour scheme of the application |  looks pleasing to my eyes
+
+|`**` |user who has very bad eyesight | change the look and feel of the application such as font size and type |  look at the font properly
+
+|`**` |user | blacklist phone numbers |  get rid of spams from other software.
+
+|`***` |celebrity manager | associate the appointment with relevant celebrities and parties |  view who are involved
+
+|`***` |celebrity manager | associate the appointment with relevant contacts |  contact the relevant parties easily
+
+|`***` |celebrity manager | associate certain appointments with payment |  know if payment is needed for next appointment
+
+|`**` |celebrity manager | consolidate payments for a certain period |   know if there is any outstanding payment
+
+|`**` |celebrity manager | group celebrities up by their different talent (acting, singing, hosting etc.) |  find suitable celebrities for event quickly.
+
+|`***` |celebrity manager | have multiple schedule calendars |  manage more than 1 different schedules in the event my client is a idol group
+
+|`*` |celebrity manager | check the celebrities I blacklisted |  organize the blacklist
+
+|`**` |celebrity manager managing a same group of celebrities| add event to the group of celebrities at the same time |  save time
+
+|`**` |celebrity manager |  obtain information or details about the upcoming schedule (show) | keep my client informed and prepared on the content
+
 |=======================================================================
 
 _{More to be added}_
```

Comment:
Keep this "More to be added" or remove?
Since the "More to be added" is removed for Use case, we may want to standardise.

Reply:
I think the more to be added is supposed to be a note can remove it